CoreFoundation/CMakeLists.txt

```diff
@@ -41,8 +41,18 @@ else()
   set(FRAMEWORK_LIBRARY_TYPE STATIC)
 endif()
 
+set(cf_swift_runtime_abi_flags)
+if(NOT CMAKE_SYSTEM_NAME STREQUAL Darwin)
```

Comment:
Can we conditionalize this on ObjC support?  I plan on trying to setup a test configuration for swift without ObjC interop on Darwin, and in that case, this should use the swift runtime ABI.